cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -521,9 +521,10 @@ makeSignedTransaction witnesses (ShelleyTxBody era txbody txmetadata) =
           (maybeToStrictMaybe txmetadata)
 
 
-makeByronKeyWitness :: NetworkId
+makeByronKeyWitness :: forall key. IsByronLegacyFormat key
```

Comment:
See how this name is misleading? Hence the suggestion for `IsByronKey`

Reply:
Yep I agree!